server/src/optima/project.py

```diff
@@ -701,17 +705,12 @@ def setData(project_id):
         reply['reason'] = 'File type of %s is not accepted!' % source_filename
         return json.dumps(reply)
 
-    project = load_project(project_id)
-    if project is None:
-        reply['reason']='Project %s does not exist.' % project_id
-        return jsonify(reply)
+    data = json.loads(file.read());
 
-    data = json.load(file)
-    data['G']['projectfilename'] = project.model['G']['projectfilename']
-    data['G']['workbookname'] = project.model['G']['workbookname']
-    data['G']['projectname'] = project.model['G']['projectname']
```

Comment:
This should work, as long as the original project was also created via Optima web application. inputpopulations and inputprograms are not set up in non-web variant, they are read from the data.